openapi-docs/README.md

```diff
@@ -127,17 +127,21 @@ Class | Method | HTTP request | Description
 ------------ | ------------- | ------------- | -------------
 *AmbianicCloudApiCollection.DefaultApi* | [**createSubscription**](docs/DefaultApi.md#createSubscription) | **POST** /subscription | Subscribe a user to Ambianic&#39;s Premium Services
 *AmbianicCloudApiCollection.DefaultApi* | [**deleteSubscription**](docs/DefaultApi.md#deleteSubscription) | **DELETE** /subscription | Delete an Ambianic&#39;s user subscription
+*AmbianicCloudApiCollection.DefaultApi* | [**getNotificationProduct**](docs/DefaultApi.md#getNotificationProduct) | **GET** /product | Retrieve notification product
```

Comment:
@vickywane in line with previous comments, this function name should be probably renamed to getProductInformation()

Reply:
An adjustment has been made to the `operationId` field in the openapi spec to correct this.

Reply:
@vickywane the change has not been reflected in the README.md file.
The path still links to `docs/DefaultApi.md#getNotificationProduct` instead of `docs/DefaultApi.md#getProductInfo`.